force-app/main/default/lwc/takeHomePayCalc/takeHomePayCalc.js

```diff
@@ -0,0 +1,557 @@
+import { LightningElement, api, wire } from "lwc";
+import { getRecord, getFieldValue } from "lightning/uiRecordApi";
+import getTaxBracketName from "@salesforce/apex/TaxMetadataController.getTaxBracketName";
+import getTaxRateForTaxBracketName from "@salesforce/apex/TaxMetadataController.getTaxRateForTaxBracketName";
+import getStartingSalaryForTaxBracketName from "@salesforce/apex/TaxMetadataController.getStartingSalaryForTaxBracketName";
+import getEndingSalaryForTaxBracketName from "@salesforce/apex/TaxMetadataController.getEndingSalaryForTaxBracketName";
+import getSocialSecurityRate from "@salesforce/apex/FederalRatesMetadataController.getSocialSecurityRate";
+import getSocialSecurityMaxVal from "@salesforce/apex/FederalRatesMetadataController.getSocialSecurityMaxVal";
+import getMedicareRate from "@salesforce/apex/FederalRatesMetadataController.getMedicareRate";
+import getStandardDeduction from "@salesforce/apex/DeductionMetadataController.getStandardDeduction";
+import SALARY_FIELD from "@salesforce/schema/Job_Application__c.Salary__c";
+import PAID_FIELD from "@salesforce/schema/Job_Application__c.How_Paid__c";
+
+const FIELDS = [SALARY_FIELD, PAID_FIELD];
+export default class takeHomePayCalc extends LightningElement {
+  @api recordId;
+  howPaidVal = null;
+  payFrequencyVal = null;
+  filingStatusVal = null;
+  salAmt = null;
+  hrsWkd = null;
+  hrlyRate = null;
+  addtlPretaxDeductAmt = 0.0;
+  addtlPosttaxDeductAmt = 0.0;
+  extraTaxPerPayPeriod = 0.0;
+  addtlIncome = 0.0;
+  yrlyTaxableInc = 0.0;
+  maxTaxBracket = "";
+  medicareRate = null;
+  socSecRate = null;
+  socSecMaxVal = null;
+  stdDeduct = null;
+  itmDeduct = 0.0;
+  error = null;
+  currYr = 0;
+  timesPerYear = 0;
+  takeHomePay = 0.0;
+  fedTax = null;
+  socSecTax = null;
+  medTax = null;
+  grossSalDiv = null;
+  fedExemptValue = null;
+  medExemptValue = null;
+  socExemptValue = null;
+  isSenior = null;
+  isBlind = null;
+  showCalculate = false;
+  showSalaried = false;
+  showHourly = false;
+  hasFilingStatus = false;
+
+
+  @wire(getRecord, { recordId: "$recordId", fields: FIELDS })
+  loadFields({ error, data }) {
+    if (error) {
+      this.error = error;
+      console.log(error);
+    } else if (data) {
+      this.howPaidVal = getFieldValue(data, PAID_FIELD);
+      this.salAmt = getFieldValue(data, SALARY_FIELD);
+      console.log('**howPaidVal: ' + this.howPaidVal);
+      console.log('**sal Amt: ' + this.salAmt);
+      if (this.howPaidVal === "Salaried Annually") {
+        this.showSalaried = true;
+      } else {
+        this.showHourly = true;
+      }
+    }
+  }
+
+  @wire(getStandardDeduction, {
```

Comment:
It might be best to have the @wire method names differ from the variable names you are assigning, i.e. `handleGetStandardDeduction()`